client/shaders/character/ROGUE/index.txt

```diff
@@ -1,2 +1,4 @@
-0 /LEFT
-1 /RIGHT
\ No newline at end of file
+0 1 /IDLE/RIGHT
+0 3 /IDLE/LEFT
+1 1 /MOVE/RIGHT
+1 3 /MOVE/LEFT
```

Comment:
can you the movement in like the rest of the characters? I guess if we don't have attack, just put it all as idle in one direction.